src/test/java/org/dita/dost/module/CleanPreprocessModuleTest.java

```diff
@@ -52,6 +52,7 @@ public void getCommonBase() {
         assertEquals(create("file:/foo/"), module.getCommonBase(create("file:/foo/a"), create("file:/foo/bar/b")));
         assertEquals(create("file:/foo/"), module.getCommonBase(create("file:/foo/bar/a"), create("file:/foo/b")));
         assertEquals(create("file:/foo/"), module.getCommonBase(create("file:/foo/bar/a"), create("file:/foo/baz/b")));
+        assertEquals(create("file:/C:/"), module.getCommonBase(create("file:/C:/a"), create("file:/D:/b")));
```

Comment:
If you revert the code change, this test doesn't test for NPE.

Reply:
You're right, this doesn't cover directly the NPE, I don't know where I can test it.